packages/exp/src/expo_commands/generate-unimodule.js

```diff
@@ -0,0 +1,148 @@
+      if (!fs.existsSync(TEMP_DIR_NAME)) {
+          fs.mkdirSync(TEMP_DIR_NAME);
+      }
+      await decompress();
+
+      fs.unlinkSync(`expo-module-template-${NPM_TEMPLATE_VERSION}.tgz`);
+      await copy(path.join(TEMP_DIR_NAME, `package`), `${configuration.jsName}`);
+      await fsExtra.remove(TEMP_DIR_NAME);
+
+      if (configuration.podName) {
+          fs.renameSync(
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate.podspec`),
+            path.join(`${configuration.jsName}`, `ios`, `${configuration.podName}.podspec`)
+          );
+
+          fs.renameSync(
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate`, `EXModuleTemplate.h`),
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate`, `${configuration.podName}.h`)
+          );
+
+          fs.renameSync(
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate`, `EXModuleTemplate.m`),
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate`, `${configuration.podName}.m`)
+          );
+
+          fs.renameSync(
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate`),
+            path.join(`${configuration.jsName}`, `ios`, `${configuration.podName}`)
+          );
+
+          fs.renameSync(
+            path.join(`${configuration.jsName}`, `ios`, `EXModuleTemplate.xcodeproj`),
+            path.join(`${configuration.jsName}`, `ios`, `${configuration.podName}.xcodeproj`)
+          );
+      } else {
+        await fsExtra.remove(path.join(`${configuration.jsName}`,`ios`));
+      }
+
+      await replace({
+        regex: "expo-module-template",
+        replacement: `${configuration.jsName}`,
+        paths:[`${configuration.jsName}`],
+        recursive: true,
+        slient: true,
+      });
+
+      await replace({
+        regex: "expo.modules.template",
+        replacement: `${configuration.javaModule}`,
+        paths:[`${configuration.jsName}`],
+        recursive: true,
+        slient: true,
+      });
+
+      await replace({
+        regex: "EXModuleTemplate",
+        replacement: `${configuration.podName}`,
+        paths:[`${configuration.jsName}`],
+        recursive: true,
+        slient: true,
+      });
+
+      await replace({
+        regex: `"version": ".*",`,
+        replacement: `"version": "1.0.0",`,
+        paths:[path.join(`${configuration.jsName}`, `package.json`)],
+        recursive: false,
+        slient: true,
+      });
+
+      const javaDir = path.join(
+        configuration.jsName,
+        'android',
+        'src',
+        'main',
+        'java',
+        ...configuration.javaModule.split('.')
+      );
+      fsExtra.mkdirpSync(javaDir);
+
+      const placeholderPath = path.join(
+        `${javaDir}`,
+        `Placeholder.java`
+      );
+      fs.appendFileSync(`${placeholderPath}`, `package ${configuration.javaModule};${os.EOL}`);
```

Comment:
I think instead of platform specific `os.EOL` (`\n` or `\r\n`) you should just use `\n` since that's the line break character the other files in the template use too.